libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -330,6 +342,13 @@ export async function fetchAndProcessPlainHtml({ url, shouldDecorateLinks = true
     path = mepFragment.target;
   }
   const res = await fetch(path.replace(/(\.html$|$)/, '.plain.html'));
+  if (res.status !== 200) {
+    lanaLog({
+      message: 'Error in fetchAndProcessPlainHtml',
+      e: `${res.statusText} url: ${res.url}`,
+      tags: 'errorType=error,module=utilities',
```

Comment:
This also seems to fall under network issues, so I would go for:
```suggestion
      tags: 'errorType=info,module=utilities',
```

Reply:
@robert-bogos 
Done.